peps/pep-0740.rst

```diff
@@ -251,12 +354,36 @@ following structure:
           "vendor-property": "foo",
           "another-property": 123
         },
-        "attestations": {
-          "some-attestation": {/* ... */},
-          "another-attestation": {/* ... */}
-        }
+        "attestations": [
+          { /* ... */ },
+          { /* ... */ }
+        ]
     }
 
+or, as pseudocode:
+
+.. code-block:: python
+
+  @dataclass
+  class Provenance:
+      version: Literal[1]
+      """
+      The provenance object's version, which is always 1.
+      """
```

Comment:
NB: This adds a `version` to the provenance object as well. Per @haydentherapper's note, this makes future extensibility with self-held keys easier (since these would presumably fall under their own fields, not under `publisher`).